internal/goutils/utils.go

```diff
@@ -0,0 +1,27 @@
+package goutils
+
+// Ptr returns a pointer to the given value.
+func Ptr[T any](input T) *T {
+	return &input
+}
+
+// MustBeNil - panics on non-empty error.
+func MustBeNil(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+// Must succeed - panics on non-empty error, returns one value.
+func Must[T any](obj T, err error) T { // nolint:ireturn
```

Comment:
Please remove Must and Must2, it is just as easy to write the following and it's way more readable.

```
a, b, err := someFunction(...)
MustBeNil(err)
```

Reply:
Ah I see your other comment that links to https://www.willem.dev/code-snippets/generic-must-function/, that's probably why you introduced these 2 functions.

The thing about the Connectors library is that we cannot be as pedantic about Go conventions as we are in our closed source repos. Future contributors will not always be well-versed in Go. So in some cases, we have to deviate about what the "Go way" of doing things is in order to make this codebase more beginner-friendly. 

If you think `MustBeNil` is confusing, then feel free to rename to `MustNotErr`, I think that will read nicely, e.g.

```
MustNotErr(loadFile("data.json"))
```

But please remove `Must` and `Must2`, they're not intuitive to someone who hasn't done a lot of Go development.

Reply:
Yes, I thought those posts had some good generalized ideas that ringed with our utils.
Your motivation is clear! I kind of didn't picture non-golang developers in a mix, which should be a mistake.